README.md

```diff
@@ -254,7 +254,15 @@ $ sfdx sgd:source:delta --from commit --ignore-destructive destructiveignore
 
 ```
 
-Note that in a situatrion where only the `--ignore [-i]` parameter is specified (and `--ignore-destructive [-D]` is not specified), then the plugin will ignore items matching `--ignore [-i]` parameter in all situations: Addition, Modification and Deletion.
+Note that in a situation where only the `--ignore [-i]` parameter is specified (and `--ignore-destructive [-D]` is not specified), then the plugin will ignore items matching `--ignore [-i]` parameter in all situations: Addition, Modification and Deletion.
+
+#### Generate a comma-separated list of the added and modified Apex classes:
+
+Depending on your testing strategy, [you may be interested in generating a a comma-separated list of the added and modified Apex classes](https://github.com/scolladon/sfdx-git-delta/issues/126) (to use in the `sfdx force:source:deploy --testlevel RunSpecifiedTests` command, for example).
+
+To cover this requirement, you can use a tool such as [yq](https://github.com/kislyuk/yq) to parse the content of the package.xml file produced by SGD:
+
+`cat package/package.xml | xq . | jq '.Package.types | select(.name=="ApexClass") | .members | join(",")'`
```

Comment:
**suggestion**(_non-blocking_): we could consider using "<" instead of cat

something like that (not tested, not sure it works this way, but the idea is here) : 
`xq . < package/package.xml | jq '.Package.types | select(.name=="ApexClass") | .members | join(",")'`

Reply:
I like your version! 
However (regardless of the use of < or cat) I've just realized that `.Package.types` won't work when there are several types in the package.xml (`.Package.types[]` would be need then). I'd have to come with a different syntax, please don't merge the PR for now.